dogs-initial/src/main/java/com/lambdaschool/dogsinitial/controller/DogController.java

```diff
@@ -17,13 +22,20 @@
 @RequestMapping("/dogs")
 public class DogController
 {
+    private static final Logger logger= LoggerFactory.getLogger(DogController.class);
+
+//    @Autowired
+//   RabbitTemplate rt;
 
```

Comment:
Why comment out the lines for sending to your create message queue? It would actually work properly.